molten-test-mockito-autoconfigure/src/main/java/org/mockito/configuration/MockitoConfiguration.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mockito.configuration;
+
+import org.mockito.exceptions.misusing.MockitoConfigurationException;
+import org.mockito.stubbing.Answer;
+
+/**
+ * Mockito configuration to set {@code com.hotels.molten.test.mockito.ReactiveMockitoConfiguration} as default.
```

Comment:
Please add this to the respective readme as well:
- What's the purpose of this?
- How to override if necessary?
- Example if needed.

Reply:
Added and fixed the related readme files in 131832b5, including the answers.